_inc/client/components/about-page/style.scss

```diff
@@ -0,0 +1,180 @@
+$color__border: #e5e5e5;
+$color__back-link: #6c7781;
+
+.jetpack-about__link-back,
+.jetpack-about__main,
+.jetpack-about__plugin {
+	background-color: $white;
+	border: 1px solid $color__border;
+}
+
+.jetpack-about__main,
+.jetpack-about__colophon {
+	font-size: 14px;
+
+		p {
+			font-size: 14px;
+		}
+}
+
+.jetpack-about__link-back {
+	padding: 16px 16px 13px;
+	border-bottom: none;
+
+	a {
+		color: $color__back-link;
+		font-size: 0.75rem;
+
+		&:hover,
+		&:active {
+			color: darken( $color__back-link, 30% );
+		}
+
+		svg {
+			fill: $color__back-link;
+			height: 24px;
+			margin-right: 4px;
+			margin-top: -3px;
+			vertical-align: middle;
+			width: 24px;
+		}
+	}
+}
+
+.jetpack-about__main {
+	padding: 36px;
+}
+
+.jetpack-about__logo {
+	width: 265px;
+}
+
+.jetpack-about__text {
+	margin-right: 40%;
+
+	@media screen and (max-width: 660px) {
+		margin-right: 0;
+	}
+
+	p {
+		font-size: 1rem;
+		line-height: 1.6;
+
+		&:last-child {
+			margin-bottom: 0;
+		}
+	}
+}
+
+.jetpack-about__images {
+	float: right;
+	text-align: center;
+	margin-bottom: 1em;
+	margin-left: 5%;
+	width: 36.5%;
+
+	@media screen and (max-width: 660px) {
+		margin-left: 0;
+		text-align: left;
+		width: 100%;
+	}
+
+	.meet-the-team {
+		max-width: 320px;
+		text-align: center;
+	}
+}
+
+.jetpack-about__gravatars {
+	display: flex;
+	flex-wrap: wrap;
+	max-width: 320px;
+
+	li {
+		margin-bottom: 0;
+		width: 25%;
+
+		img {
+			vertical-align: top;
+			width: 100%;
+		}
+	}
+}
+
+.jetpack-about__colophon {
+	margin-bottom: 3rem;
+
+	h3 {
+		margin-top: 2.4375rem;
+	}
+}
+
+.jetpack-about__services-more {
+	margin-top: 1.5em;
+	text-align: center;
+}
+
+
+.jetpack-about__services {
+	display: grid;
+	grid-template-columns: 1fr 1fr;
+	grid-gap: 16px;
+}
+
+.jetpack-about__plugin {
+	margin-bottom: 0;
+
+ 	.plugin-card-top {
+		min-height: 165px;
+		padding: 20px 20px 15px;
+		position: relative;
+
+		.name,
+		.desc,
+		.details-link {
+			margin-left: 108px;
+		}
+
+		.name h3 {
+			margin-top: 0;
+		}
+
+		.plugin-icon {
+			height: 84px;
```

Comment:
This seems to cause the plugin icons to appear smaller than on the Plugins > Add New screen:

![image](https://user-images.githubusercontent.com/426388/56652783-78398180-668c-11e9-8737-ee72e0da2cf5.png)
![image](https://user-images.githubusercontent.com/426388/56652802-7ff92600-668c-11e9-878c-0cc0bd06f3f8.png)

Reply:
This was based on the mockup, but I'd be happy to adjust larger. cc: @jeffgolenski

Reply:
Good eye Jeremy. Doesn't need to be consistent @crunnells - I think I did that in case the plugins had a lot of descriptive text, they would have more room for content.

Reply:
Fixed in 6b18501; plugin icons are now same size as core (128x128px).